relay_messages.go

```diff
@@ -132,6 +135,13 @@ func newLazyCallReq(f *Frame) lazyCallReq {
 	arg1Len := int(binary.BigEndian.Uint16(f.Payload[cur : cur+2]))
 	cur += 2
 	cr.method = f.Payload[cur : cur+arg1Len]
+
+	// arg2~2
+	cur += arg1Len
+	cr.arg2StartOffset = cur + 2
+	cr.arg2EndOffset = cur + 2 /*arg2 len*/ + int(binary.BigEndian.Uint16(f.Payload[cur:cur+2]))
```

Comment:
instead of repeating `cur + 2` to calculate the end offset, do `arg2EndOffset = cr.arg2StartOffset + [...]`